mne/io/edf/edf.py

```diff
@@ -280,6 +279,8 @@ def __init__(self, input_fname, montage, eog=None, misc=None,
 
         self.set_annotations(Annotations(onset=onset, duration=duration,
                                          description=desc, orig_time=None))
+        if montage is not None:
+            self.set_montage(montage)
```

Comment:
For completness, this was never cover and I didn't realize.
The PR modifies RawBDF.__init__ and adds a test for EDF that was already cover.

(I'll update the comment with the commit that amends it)

Reply:
addressed in #6521